### ISSUE TITLE
Files that have correctedForAccessibility set to true should not be OCRed

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/common-accessioning#1403]
See https://github.com/sul-dlss/common-accessioning/blob/main/lib/dor/text_extraction/ocr.rb#L63

We should do some more filtering here so that files with this attribute set do not show up as OCRable files.

This will prevent them from getting staged to ABBYY, and thus prevent them from being OCRed.  

It looks like we are blocking the overwriting of existing files in the cocina updater step, so this is just for efficiency.  The files would currently be OCRed, but never added to cocina.  So this would be an efficiency step, in preventing them from even being sent to OCR.  see https://github.com/sul-dlss/common-accessioning/blob/main/lib/dor/text_extraction/cocina_updater.rb#L131-L141

The logic would have to look at the filename to be OCRed, then look for the expected filename of the captioned output (e.g. .txt) and then check the `correctedForAccessibility` cocina attribute to determine if the source file should be filtered out.

**Comment by peetucket** [comment]
Update: I think we want this is done in the cocina updater class since the attributes are on the caption file and not the source file.
